Sources/SwiftSyntax/generated/SyntaxEnum.swift

```diff
@@ -534,517 +534,517 @@ public extension Syntax {
   func `as`(_ : SyntaxEnum.Type) -> SyntaxEnum {
     switch raw.kind {
     case .token: 
-      return .token(TokenSyntax(self)!)
+      return .token(TokenSyntax(self)! )
```

Comment:
This also looks like a regression that would be worth fixing to me.

Reply:
Fixed in #1218